server/service/src/processors/transfer/README.md

```diff
@@ -0,0 +1,62 @@
+# Transfers
+
+Pair records that belong to two different stores but are intrinsically linked together are called `transfers`:
+
+`RequestRequisition` -> `ResponseRequisition`
+
+`OutboundShipment` -> `InboundShipment`
+
+The term `transfer` arises from the need to `transfer` these records from one `sync site` to another, in a scenario where a pair of linked transfer records belong to two different `sites`.
+
+## Requirements
+
+From [TMF internal google doc](https://docs.google.com/presentation/d/1eEe0uBGvkXbYnKc2oLO2U0qRwFv4l0ws4QwFZa6e74s/edit#slide=id.p):
+
+![omSupply transfer workflow](./doc/omSupply_transfer_workflow.png)
+
+Big difference between mSupply transfer requirements and omSupply:
+* The inbound shipment part of the transfer is generate before outbound shipment is shipped (before inbound shipment is editable)
+* Status updates on destinaton part of transfer (response requisition and inbound shipment) are reflected on source part of the transfer (request requisition and outbound shipment)
+
+## Ownership
+
+In our synchronisation system, we have a strong rule of:
+* records can belong to **one** site only, which mean they can only be mutated on that site.
+
+This means the `other half` of transfer record needs to be generated on the site that will own that record. This is done by processing `changelog` after syncrhonisation. 
+
+## Changelog Processing
+
+Each processor (`shipment` and `requisition` transfer processor) keep track of changelog it has processed, and when it's triggerd it will try processing any records that have been changed since the last processed changelog entry.
```

Comment:
```suggestion
Each processor (`shipment` and `requisition` transfer processor) keeps track of the changelog records that it has processed, and when triggered it will try processing any records that have been changed since the last processed changelog entry.
```